db/schema.rb

```diff
@@ -308,7 +308,6 @@
     t.string "slug"
     t.string "status", default: "active"
     t.datetime "updated_at", null: false
-    t.index ["slug"], name: "index_chat_channels_on_slug", unique: true
```

Comment:
Just to be sure: should that be part of this PR? cc @rhymes

Reply:
This was removed when I ran bin/setup command. So does [this](https://github.com/thepracticaldev/dev.to/pull/7103/commits/af9233a10f446ca71a71b00fbabb4e7531de2e5a#diff-1acd2e7e27a227829d5d14a91c863bb6R1221). If these aren't needed please let me know I can push another commit to revert them.